tests/Webapi/Dom/Webapi__Dom__Navigator__test.re

```diff
@@ -0,0 +1,28 @@
+open Webapi.Dom;
```

Comment:
Thanks for writing these tests. I know it's a bit annoying, but can you also check in the compiled test output? You'll need to do:

    git add --force lib/js/tests/Webapi/Dom/Webapi__Dom__Navigator__test.js